### DISCUSSION TITLE
顶栏的消息提示，希望能够屏蔽掉自动回复，以及系统通知。

### DISCUSSION THREAD

**Discussion by kethoven** [the1812/Bilibili-Evolved#5031] [功能建议 / Ideas]
顶栏的消息提示，希望能够屏蔽掉自动回复，以及系统通知。
每次，新关注一个账号，总是会系统自动回复一个“我的消息”，就会产生消息提示。你就会想要去消除这个提示。
当时其实这样的提示是没有任何价值的。
希望能够屏蔽掉自动回复，以及系统通知。

**Comment by the1812** [comment]
顶栏只知道数字和分类，不知道消息内容，你这个应该得提给官方

**Reply by kethoven** [reply]
好的，谢谢。
